src/server/auth/server/testing/admin_test.go

```diff
@@ -928,6 +928,88 @@ func TestTokenRevoke(t *testing.T) {
 	require.Equal(t, 0, len(repos))
 }
 
+func TestRevokeTokensForUser(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	tu.DeleteAll(t)
+	defer tu.DeleteAll(t)
+
+	contains := func(tokens []*auth.TokenInfo, hashedToken string) bool {
```

Comment:
I know we use this in several tests, could you extract it out as it's own function? Could be a follow-up PR.

Reply:
good point - I'll merge and submit a follow-up PR